src/commands/account/show.js

```diff
@@ -13,11 +13,11 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import cryptography from '@liskhq/lisk-cryptography';
+import * as cryptography from '@liskhq/lisk-cryptography';
```

Comment:
Can't we have destructured `{}` import here, as we are just using a couple of functions?